x-pack/plugins/fleet/common/constants/preconfiguration.ts

```diff
@@ -58,7 +59,7 @@ export const DEFAULT_FLEET_SERVER_AGENT_POLICY: PreconfiguredAgentPolicyWithDefa
   is_default: false,
   is_default_fleet_server: true,
   is_managed: false,
-  monitoring_enabled: ['logs', 'metrics'] as Array<'logs' | 'metrics'>,
+  monitoring_enabled: Object.values(dataTypes),
```

Comment:
nitpick but can we make a `monitoringTypes` object? I think dataTypes could be augmented one day and it's probably not what we want here

Reply:
Updated in 681c371